cfme/web_ui/__init__.py

```diff
@@ -3349,7 +3349,7 @@ def select_by_value(self, value):
         index = options_map.index(value)
         if not self.is_open:
             self.open()
-        new_loc = self._loc + '/../div/ul/li[@data-original-index={}]'.format(index)
+        new_loc = self._loc + '/../div/ul/li[@data-original-index={}]/a'.format(index)
```

Comment:
I've just checked and angular select by value is only used for image buttons. I'll check the failed tests, if the cause is connected with `/a` fix.